connexion/app.py

```diff
@@ -130,12 +130,17 @@ def add_api(self, swagger_file, base_path=None, arguments=None, auth_all_paths=N
         swagger_path = swagger_path if swagger_path is not None else self.swagger_path
         swagger_url = swagger_url if swagger_url is not None else self.swagger_url
         auth_all_paths = auth_all_paths if auth_all_paths is not None else self.auth_all_paths
-        logger.debug('Adding API: %s', swagger_file)
+        logger.debug('Adding API: %s', specification)
```

Comment:
I would not print here the whole specification, it might be huge. Let's just remove this debug line for now.